synapseAnnotations/data/experimentalData.json

```diff
@@ -300,6 +300,11 @@
         "value": "surveyData",
         "description": "A data set that contains the outcome of a survey.",
         "source": "http://purl.obolibrary.org/obo/OMIABIS_0000060"
+      },
+      {
+        "value": "proteinProteinInteractionNetwork",
```

Comment:
think we decided moving forward to not use camelCase for values? So, can this be `protein-protein interaction network` which is what it is at the source?

Reply:
Because it looks nicer?  *sigh*. will change.